### ISSUE TITLE
Logs not showing up in Logs Explorer after upgrading to 0.58.1 (live logs work)

### ISSUE THREAD

**Issue by cycloptux** [SigNoz/signoz#6465]
## Bug description

After updating from 0.58.0 to 0.58.1, the Logs Explorer is not showing any log in the query window - unless I enable live logs. The window shows up as empty, saying "No logs yet. When we receive logs, they would show up here". Logs are being correctly ingested.

![2024-11-18 10_00_19-SigNoz _ Logs — Mozilla Firefox](https://github.com/user-attachments/assets/70d98729-22a6-4417-b4b0-bd41a2c29ed2)
![2024-11-18 10_00_41-SigNoz _ Live Logs — Mozilla Firefox](https://github.com/user-attachments/assets/3357dcb4-713f-4222-a6d7-fda2fb5b0025)

## Expected behavior

Logs appear and can be filtered/analyzed (as it was in 0.58.0)

## How to reproduce

N/A

## Version information 

* **Signoz version**: 0.58.1 (self-hosted through Docker Compose on a Debian 11 VM)
* **Browser version**: Firefox 132.0.2 / Edge 131.0.2903.51 (not browser-related)
* **Your OS and version**: Windows 11 (not client-related)
* **Your CPU Architecture**(ARM/Intel): AMD x86 (not client-related)

## Additional context

N/A

**Comment by cycloptux** [comment]
```INFO querier/helper.go:122 > timestamp=2024-11-18T09:59:55.775Z msg=cache misses for logs query misses=[{"Start":1731920397000,"End":1731923997000}]
INFO app/server.go:396 > timestamp=2024-11-18T09:59:55.781Z msg=/api/v3/query_range timeTaken=4 path=/api/v3/query_range
INFO utils/time.go:17 > timestamp=2024-11-18T09:59:55.786Z msg=Elapsed time func_name=GetTimeSeriesResultV3 duration=10 source=logs-explorer client=browser query=SELECT toStartOfInterval(fromUnixTimestamp64Nano(timestamp), INTERVAL 60 SECOND) AS ts, severity_text as `severity_text`, toFloat64(count(*)) as value from signoz_logs.distributed_logs_v2 where (timestamp >= 1731920397000000000 AND timestamp <= 1731923997000000000) AND (ts_bucket_start >= 1731918597 AND ts_bucket_start <= 1731923997) group by `severity_text`,ts order by value DESC viewName= servicesTab= path=/logs/logs-explorer dashboardID= alertID=
INFO app/server.go:396 > timestamp=2024-11-18T09:59:55.787Z msg=/api/v3/query_range timeTaken=16 path=/api/v3/query_range
INFO app/server.go:396 > timestamp=2024-11-18T09:59:59.736Z msg=/api/v1/health timeTaken=0 path=/api/v1/health`

**Comment by HeroesLament** [comment]
Hello, my fresh install is running into this same issue... awesome timing. If I can help just let me know.

**Comment by nityanandagohain** [comment]
Hi we have found the issue, when you select 1 hour or less than 1 hour interval it's breaking. 

We will create a new path release to address which will contain the fix https://github.com/SigNoz/signoz/pull/6470

**Comment by nityanandagohain** [comment]
please find the latest release with the fix here https://github.com/SigNoz/signoz/releases/tag/v0.58.2

**Comment by cycloptux** [comment]
> Hi we have found the issue, when you select 1 hour or less than 1 hour interval it's breaking.
> 
> We will create a new path release to address which will contain the fix #6470

Confirmed fixed when selecting >1h in 0.58.1, and completely fixed in 0.58.2. Thanks
